docs/tutorials/gke.md

```diff
@@ -151,6 +151,91 @@ gcloud container clusters create $GKE_CLUSTER_NAME \
   --workload-pool "$GKE_PROJECT_ID.svc.id.goog"
 ```
 
+### Workload Identity
+
+[Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) allows workloads in your GKE cluster to [authenticate directly to GCP](https://cloud.google.com/kubernetes-engine/docs/concepts/workload-identity#credential-flow) using Kubernetes Service Accounts
+
+You have an option to chose from using the gcloud CLI or using Terraform.
+
+=== "gcloud CLI"
```

Comment:
Allows for a tabbed view, users can click `gcloud CLI` or `terraform` opposed to a new `###` section that could confuse users

Reply:
I tested it locally. It looks good. Thanks :+1: